test/indicators/flash-message.test.js

```diff
@@ -27,14 +27,17 @@ test('FlashMessage dismiss button includes label for screenreaders', () => {
       Success!
     </FlashMessage>
   )
-  expect(dismissWrapper.find('button.dismiss').prop('aria-label')).toBeDefined()
+  expect(screen.getByRole('button')).toHaveAttribute('aria-label')
```

Comment:
You can test the `aria-label` text directly here:
```suggestion
  expect(screen.getByLabelText('Dismiss'))
```

Reply:
Thanks! I tweaked this slightly to assert `toBeInTheDocument()`. I don't love the empty chain on `expect` because if the API were to change in the testing library, this test would still pass (`expect(null)`, `expect(undefined)`, and `expect(false)` all pass!).

Reply:
That's right! I only included the first part of the assertion in the code suggestion, my bad.